install/operator/pkg/generator/assets/route/route.yml.mustache

```diff
@@ -0,0 +1,20 @@
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+    annotations:
+      console.alpha.openshift.io/overview-app-route: "true"
+    name: syndesis
+  spec:
+    host: {{Env.ROUTE_HOSTNAME}}
```

Comment:
I'm a bit confused, so this might not be an issue, should this be:
```suggestion
    host: {{.Env.ROUTE_HOSTNAME}}
```

Reply:
yeah.. the generator now support both go (.yml.tmpl) and mustache (.yml.mustache) templates.  mustache template variables don't start with '.' like the go template ones.

I find the mustache templates easier to read, but the go templates do give you more functionality if you need it.